integration/framework/scenario.go

```diff
@@ -1,41 +1,68 @@
-package framework
+package e2e
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
+	"path/filepath"
 	"strings"
 
 	"github.com/pkg/errors"
 )
 
 const (
-	MinioAccessKey = "cortex"
-	MinioSecretKey = "supersecret"
-	NetworkName    = "cortex-integration-test"
+	NetworkName        = "e2e-test"
+	ContainerSharedDir = "/shared"
 )
 
 type Scenario struct {
 	services []*Service
+
+	dir string
 }
 
 func NewScenario() (*Scenario, error) {
 	s := &Scenario{
 		services: []*Service{},
 	}
 
+	if os.Getenv("E2E_INTEGRATION_TEST_DIR") != "" {
+		s.dir = os.Getenv("E2E_INTEGRATION_TEST_DIR")
+		if err := os.MkdirAll(s.dir, os.ModePerm); err != nil {
+			return nil, err
+		}
+
+	} else {
+		tmpDir, err := ioutil.TempDir(".", "e2e_integration_test")
```

Comment:
What do you think about using `os.TempDir()` instead of `.`? Is there a good reason behind `.`?

Reply:
I was fighting with permission issues that's it ;p  `/tmp` would do.